app/volume.go

```diff
@@ -528,11 +542,24 @@ func (job *Job) waitForBackupProcessStart(timeout int) error {
 			return err
 		}
 
+		hasRunningController := false // controller is the engine process
```

Comment:
Do we need this for starting the backup? Unexpected detach should result in fail to start the backup soon.

Reply:
You are right. Fixed. Thank you